src/main.c

```diff
@@ -551,6 +552,7 @@ static gchar* r_status_formatter_shell(void)
 	gint slotcnt = 0;
 	GString *text = g_string_new(NULL);
 
+	g_string_append_printf(text, "RAUC_SYSTEM_COMPATIBLE=%s\n", r_context()->config->system_compatible);
```

Comment:
The system compatible may contains spaces (or other characters), so this needs to be shell-escaped.